apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/SpansResourceTest.java

```diff
@@ -6777,6 +6854,14 @@ void getSpanStats__whenFilterUsageLessThanEqual__thenReturnSpansFiltered(String
             getStatsAndAssert(projectName, null, filters, null, null, apiKey, workspaceName, projectStatItems);
         }
 
+        private ModelPrice randomModelPrice() {
+            return ModelPrice.values()[randomNumber(0, ModelPrice.values().length - 1)];
```

Comment:
`randomNumber` will generate an index included start and end, which will result in inclusion of `ModelPrice .DEFAULT` which does nothing, maybe would be better to exclude it?